.gitpod.yml

```diff
@@ -3,6 +3,7 @@ image: drud/ddev-gitpod-base:20220718
 tasks:
   - name: ddev-gitpod-launcher
     init: |
+      apt-get update >/dev/null && apt-get install -y ddev >/dev/null
```

Comment:
```suggestion
```
Software installed during `prebuild` will not persist in the actual workspace that opens